pkg/api/index.go

```diff
@@ -291,6 +291,17 @@ func (hs *HTTPServer) getNavTree(c *models.ReqContext, hasEditPerm bool) ([]*dto
 			Icon:        "key-skeleton-alt",
 			Url:         hs.Cfg.AppSubURL + "/org/apikeys",
 		})
+
+		// needs both feature flag and migration to be able to show service accounts
+		if hs.Cfg.IsServiceAccountEnabled() && hs.serviceAccountsService.HasServiceAccountsMigrated(c.Req.Context(), c.OrgId) {
+			configNodes = append(configNodes, &dtos.NavLink{
+				Text:        "Service accounts",
+				Id:          "serviceaccounts",
+				Description: "Create & manage Service accounts",
```

Comment:
Ampersands are discouraged in Grafana, and service accounts are not a proper noun (sv: egennamn) but rather a description, so it shouldn't be capitalized:

```suggestion
				Description: "Create and manage service accounts",
```

Alternatively, and this is mostly my personal opinion that conciseness is neat:

```suggestion
				Description: "Manage service accounts",
```

Reply:
I was following the conventions of the apikeys.

Lets do this instead 👍